docs/beta/infrequent-dashboard-support.md

```diff
@@ -0,0 +1,58 @@
+---
+id: infrequent-tier-dashboard-support
+title: Infrequent Tier Support for Dashboards
+description: Learn how to setup dashboard panels against the Infrequent data tier.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<p> <a href="/docs/beta"><span className="beta">Beta</span></a> </p>
+
+Previously, dashboards were only supported in Sumo Logic’s Continuous data tier. Now, you can also setup dashboard panels that run against the Infrequent tier.
```

Comment:
I'd rephrase it as:
Previously, dashboards were only supported in Sumo Logic's Continuous data tier. Now, you can also set up dashboard panels that run against the Infrequent tier, making it easier to include data from the Infrequent tier in the scope of dashboard panels.
```